tutorials/end_to_end/convert_params.py

```diff
@@ -0,0 +1,261 @@
+import numpy as np
+import warnings
+from scipy.optimize import fsolve
+from scipy.special import zetac
+from scipy.special import erf
+
+
+# Method to convert parameters from rate to LIF
+def convert_rate_to_lif_params(**kwargs):
```

Comment:
I would prefer it if you explicitly wrote down the accepted arguments (with type annotations) as arguments to the function, rather than accepting a dictionary of kwargs. That way, the user gets some hints about what he/she may pass to the function and what data types you are expecting.

```python
def convert_rate_to_lif_params(shape_exc: typing.Tuple[int, ...], ...) -> typing.Dict:
```
If this makes the calling too complicated, at least document the available arguments.

Reply:
Agreed, made arguments explicit it in b362062.